internal/checker/ruleset_info.go

```diff
@@ -83,46 +132,50 @@ func GetAllDownloadedRuleSets() ([]string, error) {
 	return ruleSets, nil
 }
 
-type SpitoRulesetYaml struct {
+func getRulesetConfYaml(rulesetLocation *RulesetLocation) (RulesetConfYaml, error) {
+	if err := FetchRuleset(rulesetLocation); err != nil {
+		return RulesetConfYaml{}, err
+	}
+
+	rawSpitoYaml, err := ReadRawSpitoYaml(rulesetLocation)
+	if err != nil {
+		return RulesetConfYaml{}, err
+	}
+
+	var spitoRulesYaml RulesetConfYaml
+	err = yaml.Unmarshal(rawSpitoYaml, &spitoRulesYaml)
+
+	return spitoRulesYaml, err
+}
+
+func ReadRawSpitoYaml(rulesetLocation *RulesetLocation) ([]byte, error) {
+	spitoRulesDataBytes, err := os.ReadFile(rulesetLocation.GetRulesetPath() + "/spito.yml")
```

Comment:
There is already constant  for `spito.yml` but I don't use it neither like it, because it may cause errors by not considering `.yaml` extension.
I think creating constant: `ConfigFilenameWithFullExtension` doesn't seems like good idea

Reply:
I meant two constants should be created - one for .yml and the other for .yaml filename